src/main/java/de/tum/in/www1/artemis/service/LectureUnitService.java

```diff
@@ -36,11 +36,14 @@
 import de.tum.in.www1.artemis.repository.LectureUnitCompletionRepository;
 import de.tum.in.www1.artemis.repository.LectureUnitRepository;
 import de.tum.in.www1.artemis.repository.SlideRepository;
+import de.tum.in.www1.artemis.service.connectors.pyris.PyrisWebhookService;
 
 @Profile(PROFILE_CORE)
 @Service
 public class LectureUnitService {
 
+    private final Optional<PyrisWebhookService> pyrisWebhookService;
```

Comment:
Since this is optional can you put it at the bottom of this list (also in the constructor)